digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/RoadLinkService.scala

```diff
@@ -443,6 +446,19 @@ class RoadLinkService(val vvhClient: VVHClient, val eventbus: DigiroadEventBus,
       (enrichRoadLinksFromVVH(links ++ links2, changes), changes)
     }
   }
+  //TODO: Equal method above, analise a way of overload with default values
+  def getRoadLinksAndChangesByBoundsFromVVH(bounds: BoundingRectangle, bounds2: BoundingRectangle, newTransaction: Boolean = true): (Seq[RoadLink], Seq[ChangeInfo])= {
```

Comment:
don't forget to analyse the TODO the method above "getRoadLinksAndChangesFromVVH" is equal without a parameter "newTransaction"

Reply:
It's answered in the comment below, I will remove the TODO comment